core/src/price.ts

```diff
@@ -66,7 +69,8 @@ export const calculateTransactionCollateralOutcome = function (
     const potentialOutcomeTotalPrice = potentialOutcomeCollateralAmount.multipliedBy(unitPrice); // owe
     if (
         // if owe > tab
-        potentialOutcomeTotalPrice.isGreaterThan(auction.debtDAI)
+        // soft compensation because of precision problems.
+        auction.debtDAI.minus(potentialOutcomeTotalPrice).isLessThan(new BigNumber('0.0000000000000001'))
```

Comment:
Why is this needed? If it is indeed needed, please use `toPrecision` instead https://mikemcl.github.io/bignumber.js/#toP

Reply:
I think you forgot to use second parameter to the `toPrecision` which can be `ROUND_DOWN` or `ROUND_UP`. That was the suggestion in the first place. Then you might not need to convert both numbers (otherwise the code you've did doesn't guarantee that they would be rounded to the same number and `isLessThan` wouldn't work)

Reply:
Sorry, i don't understand what precisely you mean here with:
  - not needing to convert both numbers and
  - what's the reason not to do it